### ISSUE TITLE
Create release slides for websites (SEED, Data Tools)

### ISSUE THREAD

**Issue by isalanglois** [SEED-platform/project-tracking#10]

**Comment by RDmitchell** [comment]
I am not quite sure what this issue means. 

On the SEED website the releases have different web page
https://seed-platform.org/release_notes/#version-250

Not sure I am up to doing that. 

But I could create a google doc with the same formatting and screenshots if someone else could put that into a web page. 

I have also created "slide decks" with the release notes on them, but I'm not sure how they are used. Here is an example

https://drive.google.com/file/d/1ut34_Cn94HLUaUsmGjZtCyqYPJf4NJ6E/view?usp=share_link

Is that what you are looking for?

**Comment by isalanglois** [comment]
@nllong Could you speak to this?

**Comment by nllong** [comment]
Yes, we are talking about the "slide decks". They are posted here (https://seed-platform.org/getting_started/) under the Data Guides. The website could definitely use some cleanup.

**Comment by RDmitchell** [comment]
@nllong -- yes I would really like to reorganize it and update the documentation. 

I am working on a Getting Started set for the current release, based on dev1, which has a huge number of changes. 

I will also work on a Release Notes version for that same release.

**Comment by isalanglois** [comment]
@RDmitchell Can this ticket be closed?

**Comment by RDmitchell** [comment]
Actually, I have on my list to work on the website, which is in sad need of updating. 

I would like to keep this open and put it on the FY 24 Q1 project
